knowhere/index/vector_index/ConfAdapter.cpp

```diff
@@ -98,6 +98,14 @@ ConfAdapter::CheckSearch(Config& cfg, const IndexType type, const IndexMode mode
     return true;
 }
 
+bool
+ConfAdapter::CheckRangeSearch(Config& cfg, const IndexType type, const IndexMode mode) {
+    const float DEFAULT_MIN_RADIUS = 0.0;
+    const float DEFAULT_MAX_RADIUS = std::numeric_limits<float>::max();
```

Comment:
Why we do this in function constant declaration? I guess we can do it out of the function, and we are trying to use `kSnakeCase` for constant declaration to avoid the conflict with macros

Reply:
for these kinds of constant declaration, if it only get used here. we can initiate a anonymous namespace for it in the cpp file

Reply:
These variables are only used in this API, so we declare here.
Suggest to change this style in knowhere-2.0